super-intel.c

```diff
@@ -6035,8 +6035,7 @@ static int add_to_super_imsm(struct supertype *st, mdu_disk_info_t *dk,
 			pr_err("%s controller supports Multi-Path I/O, Intel (R) VROC does not support multipathing\n",
 			       basename(cntrl_path));
 
-		if (super->hba->type == SYS_DEV_VMD && super->orom &&
-		    !imsm_orom_has_tpv_support(super->orom)) {
+		if (super->orom && !imsm_orom_has_tpv_support(super->orom)) {
```

Comment:
what about detail platform property `"3rd party NVMe :%s supported\n","` currently only for VMD will be printed

Reply:
I know. The problem is that the filed is valuable only for nvme disks (don't ask me why, it was not my decision) and should not be printed (or checked) elsewhere. Simply, I decided to not touch it then.
To honor that, I have to provide `hba_type` to `print_imsm_capability` function. If you see that better I can do this.